wetmap/src/components/contexts/selectedPictureContext.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { createContext, useState } from 'react';
+import { Photo } from '../../entities/photos';
+
+type SelectedPictureContextType = {
+  selectedPicture:    Photo
+  setSelectedPicture: React.Dispatch<React.SetStateAction<Photo>>
+};
+
+const SelectedPictureContextState = {
+  selectedPicture: {
+    id:         0,
+    created_at: '',
+    photoFile:  '',
+    label:      '',
+    dateTaken:  '',
+    latitude:   0,
+    longitude:  0,
+    month:      0,
+    UserID:     '',
+    UserName:   '',
+  },
+  setSelectedPicture: () => {},
+};
+
+export const SelectedPictureContext = createContext<SelectedPictureContextType>(
+  SelectedPictureContextState,
```

Comment:
I far as I remember there is no need to set default value for the context if context.provider is used. The consumer is always going to get value set in the provider.
It's recommended to use null as a default value for the context:

> defaultValue: The value that you want the context to have when there is no matching context provider in the tree above the component that reads context. If you don’t have any meaningful default value, specify null. The default value is meant as a “last resort” fallback. It is static and never changes over time.
https://react.dev/reference/react/createContext

Reply:
I have made the switch to use null as the alternate possible value for them